planet.cpp

```diff
@@ -51,6 +51,9 @@ Planet::Planet(float distance) {
     // Radius is from MIN_RADIUS to MAX_RADIUS
     radius = randRange(MIN_RADIUS, MAX_RADIUS);
 
+    // Find surface area and then calculate capacity based on density per area.
+    capacity = (4 * 3.14 * pow(radius, 2)) * randRange(MIN_DENSITY, MAX_DENSITY);
```

Comment:
Pi constant is defined in config.h as PI.
Also, pow(radius, 2) can be replaced by radius * radius which in most cases is most readable and more efficient.

Reply:
Very true. Could someone make the change?